app/index.html

```diff
@@ -6,7 +6,7 @@
 		<meta name="mobile-web-app-capable" content="yes">
 		<meta name="apple-mobile-web-app-capable" content="yes">
 	</head>
-	<body mol_view_root>
+	<body mol_view_root style="overflow: hidden;" >
```

Comment:
Isn't it a bad practice to add the inline styles?

Reply:
better in css, yes (but now I removed this body style because I made some changes to the layout)